hepcrawl/utils.py

```diff
@@ -261,26 +261,42 @@ def get_journal_and_section(publication):
     return journal_title, section
 
 
-def get_license(license_url='', license_text=''):
+def get_license_list(
+    license_url='',
+    license_text='',
+    license_material='',
+):
     """Get the license dictionary from the url or the text of the license.
 
     Args:
         license_url(str): Url of the license to generate.
         license_text(str): Text with the description of the license (sometimes is
             all we got...).
+        license_material(str): Material of the license.
 
     Returns:
         list(dict): license object to be added to the generated record, empty list
             if no license could be extracted.
     """
-    license = []
-    if license_url:
+    def _populate_licence_material():
```

Comment:
To avoid confusion, I'd add license as a parameter here, and pass it to the function.

```python
def _populate_license_material(license):
...

license = _populate_license_material(license)
```

That way is explicit that it uses the license you have at that point, just run locally:
```shell
python -m this | grep implicit
```